client/src/LanguageSelection.js

```diff
@@ -4,36 +4,20 @@ class LanguageSelection extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-            language: ""
+            language: "c"
         }
 
         this.handleChange = this.handleChange.bind(this);
     }
 
     handleChange(event) {
         console.log(event.target.value); 
-        if (event.target.value === "0") {
-            this.setState({
-                language: "c" 
-            }, () => {
-                console.log(this.state.language);
-            });
+        this.setState({ language: event.target.value }, () => console.log(this.state.language));
```

Comment:
refactored some of this just to dry it out a little bit - i can revert it if these changes hurt readability too much

Reply:
Looks fine to me!